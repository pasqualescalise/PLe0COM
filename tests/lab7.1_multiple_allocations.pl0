BEGIN
	new(x, int);
	new(y, int);
	new(z, short);
	x := 20;
	print x;
	y := 40;
	print y;
	z := 50;
	print z;
	x := 10;
	print x;
	new(w, byte);
	w := 5;
	print w;
	new(t, byte);
	t := 5;
	print t
END.
