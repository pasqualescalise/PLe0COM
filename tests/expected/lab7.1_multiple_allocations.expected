20
40
50
10
5
5
